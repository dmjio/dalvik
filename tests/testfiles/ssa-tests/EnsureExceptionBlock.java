class EnsureExceptionBlock {
  static String entry(Object o) {
    String s = "bar";
    try {
      s = o.toString();
    }
    catch(NullPointerException ex) {
      s = "nullstr";
    }

    return s;
  }
}
