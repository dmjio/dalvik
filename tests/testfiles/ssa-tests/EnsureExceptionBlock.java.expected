SimplePhi { numBlocks = 4
          , phis = [(0, fromList ["nullstr", "%14"])]
          , blockPhis = [(0, 2)]
          , assignedPhis = []
          }